examples/python/model-generate.py

```diff
@@ -0,0 +1,45 @@
+import onnxruntime_genai as og
+import argparse
+
+def main(args):
+    if args.verbose: print("Loading model...")
+    model=og.Model(f'{args.model}', og.DeviceType.CPU if args.execution_provider == 'cpu' else og.DeviceType.CUDA)
+    if args.verbose: print("Model loaded")
+    tokenizer=og.Tokenizer(model)
+    tokenizer_stream=tokenizer.create_stream()
+    if args.verbose: print("Tokenizer created")
+
+    if args.input is not None:
+        prompt = args.input
+    else:
+        prompt = "I like walking my cute dog"
+    input_tokens = tokenizer.encode(prompt)
+    if args.verbose: print("Prompt(s) encoded")
+
+    params = og.GeneratorParams(model)
+    params.set_search_options({"max_length": args.max_length, "top_p": args.top_p, "top_k": args.top_k})
```

Comment:
Should we add `repetition_penalty` as a search option since the GenAI config will use the default value of 1.0?